package.json

```diff
@@ -43,5 +44,15 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^8.22.0",
```

Comment:
eslint do not requires be linked in devDependencies cause react have built-in one

Reply:
this was changed in #19

Reply:
> eslint does not requires be linked in devDependencies cause react have built-in one

this was merged into master on #19